perun-core/src/main/java/cz/metacentrum/perun/core/api/ServicesManager.java

```diff
@@ -380,30 +380,24 @@
 	 *
 	 * attributes: {...hashes...}
 	 * hierarchy: {
-	 *    ** facility **
-	 *    hashes: [...hashes...]
-	 *    members: []
-	 *    children: [
-	 *      {
-	 *        ** resource1 **
-	 *        hashes: [...hashes...]
-	 *        children: []
-	 *        members: [
-	 *          {
-	 *            ** member 1 **
-	 *            hashes: [...hashes...]
-	 *          },
-	 *          {
-	 *            ** member 2 **
-	 *            ...
-	 *          }
-	 *        ]
-	 *      },
-	 *      {
-	 *        ** resource2 **
-	 *        ...
-	 *      }
-	 *    ]
+	 *   "1": {    ** facility id **
+	 *     members: {    ** all members on resource **
```

Comment:
These are all members on the facility, right?

Reply:
The same for all other places where this Javadoc is used.

Reply:
Yes, I have updated the javadocs.